content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -121,7 +121,7 @@ and installing the sidecar injector webhook configuration on the remote cluster
     $ istioctl install -f controlplane-gateway.yaml --context="${CTX_EXTERNAL_CLUSTER}"
     {{< /text >}}
 
-1. Run the following command to confirm that the ingress gateway is up and running:
+2. Run the following command to confirm that the ingress gateway is up and running:
```

Comment:
You should not have to update the numbers from `1.`. Tsis typically means you have a formatting issue.